index.d.ts

```diff
@@ -1668,6 +1672,54 @@ declare namespace Moleculer {
 		}
 	}
 
+	/**
+	 * you can use multiple modifier with a dot
+	 * e.g. `black.bgRed.bold`
+	 */
+	type KleurColor = keyof Kleur | string;
+
+	type ActionLoggerOptions = {
+		logger?: LoggerInstance;
+		logLevel?: LogLevels;
+		logParams?: boolean;
+		logResponse?: boolean;
+		logMeta?: boolean;
+		folder?: string | null;
+		extension?: string;
+		colors?: {
+			request?: KleurColor;
+			response?: KleurColor;
+			error?: KleurColor;
+		};
+		whitelist?: Array<string>;
+	};
+	type TransitLoggerOptions = {
+		logger?: LoggerInstance;
+		logLevel?: LogLevels;
+		logPacketData?: boolean;
+		folder?: string | null;
+		extension?: string;
+		colors?: {
+			receive?: KleurColor;
+			send?: KleurColor;
+		};
+		packetFilter?: Array<Packets.packetType>;
+	};
+	type CompressionOptions = {
+		method?: "deflate" | "deflateRaw" | "gzip";
+		threshold?: number | string;
+	};
+	export const Middlewares: {
```

Comment:
I apologize for sending you down the wrong path earlier.  I've subsequently learned that namespaces are appropriate for modeling CJS libraries.  `Debugging`, `Transmit`, and `Middlewares` should all be namespaces.